src/ui/mesh/qgsmeshrenderervectorsettingswidgetbase.ui

```diff
@@ -639,26 +706,16 @@
   <tabstop>mLineWidthSpinBox</tabstop>
   <tabstop>mColoringMethodComboBox</tabstop>
   <tabstop>mColorWidget</tabstop>
-  <tabstop>mColorRampShaderMinimumEditLine</tabstop>
-  <tabstop>mColorRampShaderMaximumEditLine</tabstop>
   <tabstop>mColorRampShaderLoadButton</tabstop>
-  <tabstop>mMinMagLineEdit</tabstop>
-  <tabstop>mMaxMagLineEdit</tabstop>
   <tabstop>mDisplayVectorsOnGridGroupBox</tabstop>
```

Comment:
@elpaso, can you please add the new widgets and fix the tabstop in the .ui files? thanks.

Reply:
@DelazJ sorry but I don't know what you are talking about :)

Reply:
This tabstops section lists the widgets in the order they will be accessed by users using their keyboard (the TAB key) to navigate through the GUI. So missing items are put to the end, hence an unordered and confusing navigation.
To fix it, open the ui in Qt Designer, Edit menu --> Edit tab order: you probably see that numbers next to the widgets are not in the correct order. click or right-click and you get access to ways to fix.

Reply:
@DelazJ thanks, I know what tabstops are, I just didn't recall changing them, the order probably broke when I replaced the widgets.

Reply:
@elpaso Sorry for the explanation, then. It's unfortunately very frequent that when changes are done to ui, people do not fix that. and yes, Designer is the culprit as it places by default new items at the end regardless their real placement in the GUI.

Reply:
@DelazJ fixed, thank you for checking this PR.